xml/System.Threading.Tasks/TaskCompletionSource.xml

```diff
@@ -117,6 +118,7 @@ The <xref:System.Threading.Tasks.Task> created by this instance and accessible t
         <param name="creationOptions">The options to use when creating the underlying <see cref="T:System.Threading.Tasks.Task" />.</param>
         <summary>Creates a <see cref="T:System.Threading.Tasks.TaskCompletionSource" /> with the specified state and options.</summary>
         <remarks>To be added.</remarks>
+        <exception cref="T:System.ArgumentOutOfRangeException">The <paramref name="creationOptions" /> represent options invalid for use with a <see cref="T:System.Threading.Tasks.TaskCompletionSource" />.</exception>
```

Comment:
```suggestion
        <exception cref="T:System.ArgumentOutOfRangeException"><paramref name="creationOptions" /> represents options that are invalid for use with a <see cref="T:System.Threading.Tasks.TaskCompletionSource" />.</exception>
```

Reply:
Maybe:
```xml
One or more of the specified <paramref name="creationOptions" /> are invalid for use with a <see cref="T:System.Threading.Tasks.TaskCompletionSource" />.
```

If a change is made the same change should be made above as well.

Reply:
I like @kouvel 's suggestion, it sounds similar to how we have worded similar exceptions for enums out of range in other places.

```suggestion
        <exception cref="T:System.ArgumentOutOfRangeException">One or more of the specified <paramref name="creationOptions" /> are invalid for use with a <see cref="T:System.Threading.Tasks.TaskCompletionSource" />.
</exception>
```